README.md

```diff
@@ -17,6 +17,16 @@ or by running:
 ```
 cordova plugin add cordova-plugin-firebase --save
 ```
+or to hide notifications when app is in foreground by adding:
```

Comment:
Add this info under the `Configuring Notifications` section

Reply:
After adding it in `Configuring Notifications` should I keep the change is the `README` or should I just remove it from there ?